Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -2277,6 +2277,11 @@ return
 			},
 			Variables =
 			{
+				m_BlockHeight =
+				{
+					Type = "number",
+					Notes = "The block's hitbox height.",
```

Comment:
I am not sure why exported, I think it can be removed from export.
It returns a value between 0.5 to 1.0, but for many blocks that are smaller or equal 0.5 it returns 1.